### PR TITLE
Update download desktop page

### LINKED ISSUE #2867: Implement copy updates on /download/desktop/ - 18.04
* Update copy and links with the [copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)
* Remove the latest download from the html, not the copy doc

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Updated page inline with [copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/download/desktop>
- See that page matches [copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#)

## Issue / Card

Fixes #2867

### CODE REVIEW

File: templates/download/desktop/index.html [hunk 1]
```diff
@@ -38,6 +38,9 @@ <h2>Ubuntu {{lts_release_full_with_point}}</h2>
             </script>
           </p>
           <p class="p-card__content"><a href="/download/alternative-downloads">Alternative downloads and torrents&nbsp;&rsaquo;</a></p>
+          <p>
+            <small>For other versions of Ubuntu Desktop including torrents, the network installer, a list of local mirrors, and past releases. <a href="/download/alternative-downloads">See our alternative downloads</a></small>
```
Comment (pmahnke) [inline templates/download/desktop/index.html]: @steverydz the cta should be a new paragraph with  a non-breaking space and '>'

**Comment by steverydz** [comment]
@pmahnke Done

**Comment by frankiegranato** [comment]
Looking good aside from one small thing. Can you close up the spacing between the "For other versions .... " text and the "See our alternative downloads" link please?
(as indicated in the
download ubuntu desktop - amendment
 attached screenshot)

**Comment by frankiegranato** [comment]
Thanks @steverydz

**Pull request merged**